cypress/e2e/about.cy.ts

```diff
@@ -51,9 +59,17 @@ describe('About page', () => {
     })
 
     context('Portrait mode', () => {
-        beforeEach(() => {
-            cy.viewport('iphone-5')
+        before(() => {
             cy.visit('/about')
+            // This wait time is to give the page time to load from Prod when ran in CI.
+            cy.wait(3000)
+        })
+
+        beforeEach(() => {
+            // The resolution is in the beforeEach() instead of before() to
+            // prevent Cypress from defaulting to other screen sizes between tests.
+            cy.viewport(640, 1136)
```

Comment:
can you add a comment to explain this resolution

Reply:
can you add the same to the other ones too? ♻️

Reply:
@ermish Good call! I added those comments in the latest fixup commits.